media/videostitcher/stitcher_test.go

```diff
@@ -42,15 +47,15 @@ const (
 // *   Video Stitcher API
 // *   Cloud Resource Manager API (needed for project number translation)
 
-// TestStitcher tests major operations on slates, CDN keys, VOD sessions, and
+// TestMain tests major operations on slates, CDN keys, VOD sessions, and
```

Comment:
Nit: can probably remove this comment; TestMain is just for setup/teardown of resources.

Reply:
Removed comment